src/tools/ScenarioMeasurement/Util/ProcessHelper.cs

```diff
@@ -52,6 +53,16 @@ public ProcessHelper(Logger logger)
             psi.Arguments = Arguments;
             psi.WorkingDirectory = WorkingDirectory;
             psi.UseShellExecute = GuiApp;
+
+            if (EnvironmentVariables != null)
+            {
+                foreach (KeyValuePair<string, string> pair in EnvironmentVariables)
```

Comment:
good spot for `var`.
```suggestion
                foreach (var pair in EnvironmentVariables)
```

Reply:
because 1) the variable is only used locally and  2) it's more convenient to do declare the implicit type?